model/Software/Properties/contentIdentifier.md

```diff
@@ -4,11 +4,14 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-Provides a place to record the canonical, unique, immutable identifier for each software artifact using the artifact's gitoid.
+Used by SPDX producers to record the artifact’s gitoid: a canonical, unique, immutable identifier that can be used for software integrity verification.
```

Comment:
Obviously is not only a "gitoid", but any content identifier.

Reply:
It may be more appropriate to include an identifier tag in the syntax, like IANA uses to qualify identifier types - similar to what SPDX does today : dns:www.foo.com  SHA256:.......

Reply:
There are other content identifiers out there, as Alexios points out.    Let's handle the broadening as a separate PR though, and move this forward.    The original had gitoid in it, and is just restructured here.